iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HMSTablePropertyHelper.java

```diff
@@ -131,6 +151,103 @@ public static void updateHmsTableForIcebergTable(
     tbl.setParameters(parameters);
   }
 
+  /**
+   * Create {@link PartitionSpec} based on the partition information stored in
+   * {@link TransformSpec}.
+   * @param configuration a Hadoop configuration
+   * @param schema iceberg table schema
+   * @return iceberg partition spec, always non-null
+   */
+  public static PartitionSpec createPartitionSpec(Configuration configuration, Schema schema) {
```

Comment:
I think It should be in `IcebergTableUtil`

Reply:
I need it in HiveIcebergRESTCatalogClientAdapter which is in iceberg-catalog module. IcebergTableUtil is in iceberg-handler module which references iceberg-catalog, so I can not reference IcebergTableUtil from HiveIcebergRESTCatalogClientAdapter, since it would add circular dependency.